packages/dashboard-frontend/src/components/Loader/Step/buildSteps.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2018-2021 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { List, LoaderStep, IdeLoaderSteps } from './index';
+
+export function buildSteps(flow: 'ide' | 'factory' = 'ide'): List<LoaderStep> {
```

Comment:
I think we should not add 'factory' until implementing this functionality because implementation includes only 'ide' flow